src/prommis/uky/uky_flowsheet.py

```diff
@@ -1739,53 +1748,75 @@ def initialize_system(m):
     seq.set_guesses_for(m.fs.rougher_org_make_up.outlet, tear_guesses8)
     seq.set_guesses_for(m.fs.cleaner_org_make_up.outlet, tear_guesses9)
 
```

Comment:
Above - this is a lot of tear streams. Do we actually need that many for this system? How many tear streams did the SM tool identify>

Reply:
Not all of these were necessary. I've narrowed it down to just the leaching unit and the SX units. Maybe the latter will be resolved with an `SXInitializer`?